File: app/src/main/java/com/example/aqqhome/QuanLyActivity.java
package com.example.aqqhome;

import androidx.appcompat.app.AppCompatActivity;
import androidx.cardview.widget.CardView;

import android.content.Intent;
import android.os.Bundle;
import android.widget.ImageView;

public class QuanLyActivity extends AppCompatActivity {
    private ImageView cardView,back;
    private CardView suachiphi,xemthongtin;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_quanly);
        cardView = findViewById(R.id.themtoanha);
        suachiphi = findViewById(R.id.suachiphi);
        xemthongtin = findViewById(R.id.xemthongtin);
        cardView.setOnClickListener(v -> {
            startActivity(new Intent(QuanLyActivity.this, AddRoomActivity.class));
        });
        suachiphi.setOnClickListener(v -> {
            startActivity(new Intent(QuanLyActivity.this, Admin_SuachiphiActivity.class));
        });
        xemthongtin.setOnClickListener(v -> {
            startActivity(new Intent(QuanLyActivity.this, Admin_XemthongtinActivity.class));
        });
        back = findViewById(R.id.back);
        back.setOnClickListener(v -> {
            startActivity(new Intent(QuanLyActivity.this, AdminActivity.class));
        });
    }

}

File: app/src/main/java/com/example/aqqhome/admin/QuanLyActivity.kt
package com.example.aqqhome.admin

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity
import androidx.cardview.widget.CardView
import com.example.aqqhome.R

class QuanLyActivity : AppCompatActivity() {
    private lateinit var cardView: ImageView
    private lateinit var back: ImageView
    private lateinit var suachiphi: CardView
    private lateinit var xemthongtin: CardView
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_quanly)
        cardView = findViewById(R.id.themtoanha)
        suachiphi = findViewById(R.id.suachiphi)
        xemthongtin = findViewById(R.id.xemthongtin)
        cardView.setOnClickListener(View.OnClickListener { v: View? ->
            startActivity(
                Intent(
                    this@QuanLyActivity,
                    AddRoomActivity::class.java
                )
            )
        })
        suachiphi.setOnClickListener(View.OnClickListener { v: View? ->
            startActivity(
                Intent(
                    this@QuanLyActivity,
                    Admin_SuachiphiActivity::class.java
                )
            )
        })
        xemthongtin.setOnClickListener(View.OnClickListener { v: View? ->
            startActivity(
                Intent(
                    this@QuanLyActivity,
                    Admin_XemthongtinActivity::class.java
                )
            )
        })
        back = findViewById(R.id.back)
        back.setOnClickListener(View.OnClickListener { v: View? ->
            startActivity(
                Intent(
                    this@QuanLyActivity,
                    AdminActivity::class.java
                )
            )
        })
    }
}